src/components/POLine/POLineVersionView/DonorsVersionView/DonorsVersionView.js

```diff
@@ -0,0 +1,53 @@
+import PropTypes from 'prop-types';
+import {
+  useCallback,
+  useContext,
+  useMemo,
+} from 'react';
+
+import {
+  DonorsListContainer,
+  VersionViewContext,
+} from '@folio/stripes-acq-components';
+import { TextLink } from '@folio/stripes/components';
+
+import { getDonorUrl } from './utils';
+
+export const DonorsVersionView = ({ version }) => {
+  const donorOrganizationIds = version?.donorOrganizationIds;
+  const versionContext = useContext(VersionViewContext);
+
+  const changedDonorIds = useMemo(() => {
+    const changedDonors = versionContext?.changes?.filter((change) => {
+      return change.path.startsWith('donorOrganizationIds');
+    });
+    const donorIds = changedDonors.map(({ values }) => values.filter(Boolean));
+
+    return donorIds.flat();
+  }, [versionContext?.changes]);
+
+  const renderKeyValueComponent = useCallback(({ value, id, asLink = false }) => {
+    const content = asLink ? <TextLink to={getDonorUrl(id)}>{value}</TextLink> : value;
+    const isUpdated = changedDonorIds?.includes(id);
+
+    const displayValue = isUpdated ? <mark>{content}</mark> : content;
```

Comment:
there is util in `stripes-acq-component` to prepare formatted adapted for version history usage - https://github.com/folio-org/stripes-acq-components/blob/master/lib/VersionHistory/hooks/useVersionWrappedFormatter/useVersionWrappedFormatter.js#L6

e.g. of usage https://github.com/folio-org/stripes-acq-components/blob/master/lib/FundDistribution/FundDistributionView/FundDistributionView.js#L101-L105